# Configuration file for the microphone
lex:
  default_param:
    user_id : "micol_testing"
    bot_name: "QTDemo"
    bot_alias: "qt_sample_demo" 
    region_name: "us-west-2"
  chris_param:
    user_id : "chris_testing"
    bot_name: "GreeterPrototype"
    bot_alias: "test_bot" 
    region_name: "us-west-2"


google:
  default_param:
    project_id : "agent1-ewhnt"
    language: "it"
    session_id: "test_bot" 